core/src/apps/bitcoin/sign_tx/layout.py

```diff
@@ -62,6 +62,7 @@ async def confirm_output(
     coin: CoinInfo,
     amount_unit: AmountUnit,
     output_index: int,
+    chunkify: bool = False,
```

Comment:
remove default value

Reply:
Done in [7e03759](https://github.com/trezor/trezor-firmware/pull/3237/commits/7e03759ea7c3f0e2e56218ee1bd5cb18bef9c134)